cmd/kubeapps-apis/plugins/fluxv2/packages/v1alpha1/cache.go

```diff
@@ -320,20 +330,76 @@ type fetchValueJob struct {
 }
 
 type fetchValueJobResult struct {
-	result interface{}
-	err    error
+	key   string
+	value interface{}
+	err   error
+}
+
+// return all keys, optionally matching a given filter (repository list)
```

Comment:
I didn't want to use the word "repo" anywhere in that file, because this is supposed to have been a cache usable for storing all kinds of different things, right?

Reply:
I will leave a comment in the doc as you asked